ui/pages/Project/components/ProjectCollaborators.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import orderBy from 'lodash/orderBy'
+
+import { Table, Popup } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+import { getProject } from 'redux/rootReducer'
+
+
+const ProjectCollaborators = props => (
+  <Table className="noBorder">
+    <Table.Body className="noBorder">
+      {
+        orderBy(props.project.collaborators, [c => c.hasEditPermissions, c => c.email], ['desc', 'asc']).map((c, i) =>
+          <Table.Row key={c.email} className="noBorder">
+            <Table.Cell style={{ padding: '0px' }} className="noBorder">
+              {c.displayName ? `${c.displayName} ▪ ` : null}
+              {
+                 c.email ?
+                   <i><a href={`mailto:${c.email}`}>{c.email}</a></i> : null
+              }
+
+            </Table.Cell>
+            <Table.Cell style={{ padding: '2px 10px', textAlign: 'center', verticalAlign: 'top' }} className="noBorder">
+              <Popup
+                position="top center"
+                trigger={<b style={{ cursor: 'pointer' }}> {c.hasEditPermissions ? ' † ' : ' '}</b>}
+                content={"Has 'Manager' permissions"}
+                size="small"
```

Comment:
extra line-breaks appearing in collaborator lists. 
Example showing only 1st letter:

![image](https://user-images.githubusercontent.com/6240170/40095330-6b11c5f0-5899-11e8-914c-aaca34b20614.png)